### PR TITLE
Fix bugs : PullRefresh following + Start Screen 🌹🌷 

### PR DESCRIPTION
**Pull request by loicmisenta**
This PR add the following : 
- Pull refresh now working for following 
- Pull refresh now working when it's empty and we swipe down
- Start Screen position of the image is lower to keep space for text

image

### CODE REVIEW

File: app/src/main/java/com/example/triptracker/view/home/HomeScreen.kt [hunk 1]
```diff
@@ -351,9 +351,11 @@ fun DisplayItineraries(
     homeViewModel: HomeViewModel,
     test: Boolean = false,
     tabSelected: HomeCategory = HomeCategory.TRENDING,
+    userProfileViewModel: UserProfileViewModel = viewModel()
 ) {
   var goodPadding = PaddingValues(0.dp, 0.dp, 0.dp, 70.dp)
-  val usermail = AmbientUserProfile.current.userProfile.value.mail
+  val userProfile = AmbientUserProfile.current.userProfile.value
+  val usermail = userProfile.mail
```
Comment (theoschiff) [inline app/src/main/java/com/example/triptracker/view/home/HomeScreen.kt:358]: Why did you split this line in two if you do not use the userProfile variable ? 
Could you please only keep a single line? 
Thank you

File: app/src/main/java/com/example/triptracker/view/home/HomeScreen.kt [hunk 2]
```diff
@@ -480,17 +482,32 @@ fun HomePager(
...
+                          color = MaterialTheme.colorScheme.onBackground,
+                          fontFamily = FontFamily(Font(R.font.montserrat_regular)))
+                    },
+                    isRefreshing = isRefreshing.value,
+                    onRefresh = {
+                      isRefreshing.value = true
+                      homeViewModel.fetchItineraries() {
+                        isRefreshing.value = false
+                        homeViewModel.filterByFollowing(userEmail)
+                      }
```
Comment (theoschiff) [inline app/src/main/java/com/example/triptracker/view/home/HomeScreen.kt:508]: Thanks for adding this ! I did not catch it in your previous PR !
Nice job

File: app/src/main/java/com/example/triptracker/view/map/MapPopups.kt [hunk 3]
```diff
@@ -401,7 +401,7 @@ fun StartScreen(
                                               start = 10.dp,
                                               end = 10.dp,
                                               bottom = 0.dp)
-                                          .size(screenWidth, screenHeight * 0.2f)) {
+                                          .size(screenWidth, screenHeight * 0.3f)) {
```
Comment (theoschiff) [inline app/src/main/java/com/example/triptracker/view/map/MapPopups.kt:404]: You're picky, but in a good way 🥰

File: app/src/main/java/com/example/triptracker/viewmodel/HomeViewModel.kt [hunk 4]
```diff
@@ -338,6 +338,8 @@ class HomeViewModel(
 
   /** Filter the itinerary posted by your following users */
   fun filterByFollowing(usermail: String) {
+    UserProfileViewModel().fetchAllUserProfiles { userProfileList = it }
+    val testUser = userProfileList.firstOrNull { it.mail == usermail } ?: dummyProfile
```
Comment (theoschiff) [inline app/src/main/java/com/example/triptracker/viewmodel/HomeViewModel.kt:342]: great! now can refresh the following view

**Review by theoschiff — APPROVED** [review]
Good PR! Just a minor change to do!

File: app/src/main/java/com/example/triptracker/view/home/HomeScreen.kt (hunk 1 above)
Comment (loicmisenta) [reply app/src/main/java/com/example/triptracker/view/home/HomeScreen.kt:358]: Yes, of course ! 💮🌷

**Pull request merged**